docs/data/base/components/input/InputAdornments.js

```diff
@@ -60,7 +60,16 @@ export default function InputAdornments() {
   };
 
   return (
-    <Box sx={{ display: 'flex', '& > * + *': { ml: 1 } }}>
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 2,
+        '@media (min-width: 600px)': {
+          flexDirection: 'row',
+        },
```

Comment:
```suggestion
        display: 'flex',
        flexDirection: {xs: 'column', sm: 'row'},
        gap: 2,
```

Instead of using media queries directly in the Box `sx` prop, you can write responsive styles like so!